src/openforms/registrations/contrib/stuf_zds/client.py

```diff
@@ -287,6 +287,43 @@ def create_zaak_document(
 
         return None
 
+    def create_zaak_attachment(
+        self, zaak_id: str, doc_id: str, submission_attachment: SubmissionFileAttachment
+    ) -> None:
+        """
+        Create a zaakdocument with the submitted file.
+        """
+        template = "stuf_zds/soap/voegZaakdocumentToe.xml"
+
+        submission_attachment.content.seek(0)
+
+        base64_body = base64.b64encode(submission_attachment.content.read()).decode()
+
+        context = self._get_request_base_context()
+        context.update(
+            {
+                "zaak_identificatie": zaak_id,
+                "document_identificatie": doc_id,
+                "titel": "bijlage",
+                "auteur": "open-forms",
+                "taal": "nld",
+                "inhoud": base64_body,
+                "status": "definitief",
+                "bestandsnaam": submission_attachment.get_display_name(),
+                "formaat": submission_attachment.get_format(),
+                "beschrijving": "Bijgevoegd document",
```

Comment:
Fine for now, but I expect in the future there will be meta-data on the form field that shall serve as input for the attributes here, such as the label of the form field etc.